zerver/management/commands/delete_old_unclaimed_attachments.py

```diff
@@ -76,7 +76,12 @@ def clean_attachment_upload_backend(self, dry_run: bool = True) -> None:
         cutoff = timezone_now() - timedelta(minutes=5)
         print(f"Removing extra files in storage black-end older than {cutoff.isoformat()}")
         to_delete = []
-        for path_id, modified_at in all_message_attachments():
+        for file_path, modified_at in all_message_attachments(thumbnails=True):
+            if file_path.startswith("thumbnails/"):
+                path_parts = file_path.split("/")
+                path_id = "/".join(path_parts[1 : len(path_parts) - 1])
+            else:
+                path_id = file_path
```

Comment:
Would it make sense for this calculation of "Given a thumbnail path, get the original file path" to be a function, perhaps one exposed by `upload_backend`? It would be nice to have tests on that logic, at least.